.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+---
+label: 'needs review'
+---
+
+<!---
+Thank you for your pull request. Please provide below the name of the app, a short description 
+and a link to it's repository or homepage. Also, tell us why you think it's awesome!
+
+Make sure the PR adheres to our contribution guidelines:
+https://github.com/agarrharr/awesome-cli-apps/blob/master/contributing.md
+-->
+
+#### New App Submission
```

Comment:
Change to "New CLI Submission"?
As that is the focus of the repo.

Reply:
While most people that are going to be making submissions here will probably understand what CLI means... Not all will refer to their submission as such. App is a more commonly understood term, and is even referenced in this title of this repo.